errors/no-script-in-document-page.md

```diff
@@ -1,27 +0,0 @@
-# Script component inside \_document.js
```

Comment:
I don't think we want to delete this as older Next.js versions that link to this will 404